docs/architecture.md

```diff
@@ -98,6 +98,7 @@ spec:
             value: GET
 
     - name: my-opa-policy
+      priority: 1 # optional - causes this policy to be evaluated after other policies of higher priority, where 0 is the highest and the default; can be used in configs of any phase (identity, metadata, authorization, response) to set priority of execution within the phase
```

Comment:
Might be that this is commonly used but wouldn't priority first/last be easier to understand?

Reply:
0 (zero) has to be the default and the highest priority, i.e. the one that goes first; otherwise we would have to chose an arbitrary integer number to be the default (e.g. 9999), one that is sufficiently high to accommodate as many other (lower) numbers as we one might need to set other blocks of evaluators to run consecutively. Virtually, there is no such high integer number that wouldn't set a limit to the number of blocks of lower priority (unless negative values are also supported). In general, I'd say an inverted priority scale where the lower the number, the later it runs overcomplicates both implementation and documentation.

On top of that, there's what you said. Zero for the highest priority is commonly used. So I would stick with this convention and would not change anything here.